v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/transformer/SourceRowToMutationDoFn.java

```diff
@@ -73,7 +80,9 @@ public void processElement(ProcessContext c) {
       c.output(mutation);
     } catch (Exception e) {
       // TODO: Add DLQ integration once supported.
-      LOG.error("Unable to transform source row to spanner mutation: {}", e.getMessage());
+      LOG.error("Unable to transform source row to spanner mutation: {} {}", e, e.getMessage());
+      e.printStackTrace();
```

Comment:
please remove printstacktrace

Reply:
But that helps with debugging, thats what helped find the null ptr exception.